recipes/ssubmit/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "ssubmit" %}
+{% set version = "1.0.0" %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mbhall88/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: da6379d478b817cefce625d783f946884e035edbf1d4c11727d3f9d86756def9
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}              # [not win]
+    - {{ stdlib("c") }}                # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
```

Comment:
Have you tried using the normal `c` compiler on Windows?

Reply:
No, to be honest, I essentially just copied this from ripgrep's recipe. I'll give it a go

Reply:
I removed windows as this tool submits jobs to slurm's sbatch system and slurm is a linux utility, so windows doesn't really make sense anyway